In [35]:
import pandas as pd

neiss_df = pd.read_csv('NEISS2014.csv')

# Get data frame where the narrative contains the word skateboard.
skateboard_df = neiss_df[neiss_df['narrative'].str.contains("skateboard", case=False)]

# Get the number of accidents involving a skateboard
nb_skateboard_accidents = len(skateboard_df.index)
print("The number of injuries that involved a skateboard: %s" % nb_skateboard_accidents)

The number of injuries that involved a skateboard: 466


In [56]:
# Get the number of injuries that involved a Male
nb_male_accidents = len(skateboard_df.loc[skateboard_df['sex'] == "Male"].index)
print("Percentage Male incidents: {:.1%}".format(nb_male_accidents/nb_skateboard_accidents))
print("%s Incidents" % nb_male_accidents)

Percentage Male incidents: 82.4%
384 Incidents


In [57]:
# Get the number of injuries that involved a Female
nb_female_accidents = len(skateboard_df.loc[skateboard_df['sex'] == "Female"].index)
print("Percentage Female incidents: {:.1%}".format(nb_female_accidents/nb_skateboard_accidents))
print("%s Incidents" % nb_female_accidents)

Percentage Female incidents: 17.6%
82 Incidents


In [51]:
# Sum the age of patients
average_age = skateboard_df['age'].mean()
print("The average age of someone injured in an incident involving a skateboard: %0.2f years old" % average_age)

The average age of someone injured in an incident involving a skateboard: 17.99 years old


# Results

The number of injuries that involved a skateboard: **466**

---

Percentage skateboard incidents Female/Male:

Sex | Percentage | Nb incidents
--- | --- | ---
Female | 17.6% | 82
Male | 82.4% |  384

---

The average age of someone injured in an incident involving a skateboard: **17.99 years old**